src/app/check-auth.js

```diff
@@ -53,21 +51,22 @@ const checkLoggedIn = (state, wsControllerURL) => {
       wsControllerURL: The full controller websocket url that the controller
         is stored under in redux in order to determine it's logged in status.
 */
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
Adding a name here made the middleware go funky so I skipped the lint. Something to look into later.

Reply:
Is this tracked by an issue?